crates/rpc/rpc/src/result.rs

```diff
@@ -18,21 +18,21 @@ pub(crate) trait ToRpcResult<Ok, Err> {
     fn map_rpc_err<'a, F, M>(self, op: F) -> RpcResult<Ok>
     where
         F: FnOnce(Err) -> (i32, M, Option<&'a [u8]>),
-        M: Into<String>;
+        M: ToString;
```

Comment:
please revert to `Into<String>`

we want `Into<String>` here because `ToString` would allocate an additional message